bazel/rules_foreign_cc.patch

```diff
@@ -1,13 +1,25 @@
 diff --git a/foreign_cc/built_tools/pkgconfig_build.bzl b/foreign_cc/built_tools/pkgconfig_build.bzl
-index 64cb677..d654f67 100644
+index 64cb677..9a8c62c 100644
 --- a/foreign_cc/built_tools/pkgconfig_build.bzl
 +++ b/foreign_cc/built_tools/pkgconfig_build.bzl
-@@ -13,7 +13,7 @@ load("//toolchains/native_tools:tool_access.bzl", "get_make_data")
+@@ -9,6 +9,10 @@ load(
+     "built_tool_rule_impl",
+ )
+ load("//toolchains/native_tools:tool_access.bzl", "get_make_data")
++load(
++    "//foreign_cc/private/framework:platform.bzl",
++    "os_name",
++)
+ 
  def _pkgconfig_tool_impl(ctx):
      make_data = get_make_data(ctx)
-     script = [
--        "./configure  --with-internal-glib --prefix=$$INSTALLDIR$$",
-+        "CFLAGS=-Wno-int-conversion CXXFLAGS=-Wno-int-conversion ./configure  --with-internal-glib --prefix=$$INSTALLDIR$$",
-         "%s" % make_data.path,
+@@ -18,6 +22,9 @@ def _pkgconfig_tool_impl(ctx):
          "%s install" % make_data.path,
      ]
+ 
++    if os_name(ctx) == "macos":
```

Comment:
```suggestion
+    if os_name(ctx) == "macos" and ctx.platform_has_constraint(cpu = "arm64"):
```

im thinking this doesnt matter _that_ much as its just for compiling pkg_config afaict but im also thinking the patch is more likely to be accepted upstream if its targetted to need

Reply:
same issue on intel mac with xcode 15.3